wireshark/README.md

```diff
@@ -0,0 +1,56 @@
+# How to analyze CFLIB CRTP traffic using Wireshark
```

Comment:
Shouldn't this be part of the CFlib doc folder? it would be good to have all documentation at on place.